src/sentry/features/__init__.py

```diff
@@ -153,7 +153,7 @@
 default_manager.add("organizations:slack-overage-notifications", OrganizationFeature, True)
 default_manager.add("organizations:streamline-targeting-context", OrganizationFeature, True)
 default_manager.add("organizations:symbol-sources", OrganizationFeature)
-default_manager.add("organizations:team-roles", OrganizationFeature, True)
+default_manager.add("organizations:team-roles", OrganizationFeature)
```

Comment:
Are we also blocked on creating a feature handler for this? I tried to find one in the `getsentry` repo but couldn't find one

Reply:
No, talked to @RyanSkonnord in slack and they're fine disabling the feature for now, see latest messages in #discuss-enterprise